include/escalus.hrl

```diff
@@ -27,3 +27,11 @@
         event_client :: any(),
         props :: list()
        }).
+
+-record(extaddress, {
```

Comment:
Maybe I'm limited or sth but I think that using a map here would simplify developer's life when it comes to debugging (which is quite common part of our day to day life :) ). The map could have it's own type with required and optional type.

Reply:
I have nothing against maps, they are so useful when you need flexibility. But when I know for sure what the data structure is supposed to contain I prefer records, they make code so much more readable.

Reply:
> they make code so much more readable

I'd say the same about maps :) plus they make debugging so much nicer :)

Reply:
Do they give you code completion?

Reply:
This case is not a big deal, so if it is your strict requirement then fine, I can change it to a map. But, really, we should give it a serious though. From what I know, our coding guidelines do not say "never use records", but the practice seems to be heading in that direction. I'd suggest we make some effort to formulate a common approach, stating which data structure should/can be applied in which cases, to avoid such tensions in the future.

Reply:
It's not a strict requirement, note that I didn't request for changes, just commented in the review  :)

Reply:
Ok, got it. Can it be merged, then?